pkgs/development/tools/cmake-format/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, autopep8
+, flake8
+, jinja2
+, pylint
+, pyyaml
+}:
+
+buildPythonApplication rec {
+  pname = "cmake_format";
```

Comment:
I see upstream use two naming formats, `cmake_format` and `cmake-format`. I think for Nix, the most important thing is that the attrname and pkgname should match (currently they don't).

Reply:
I changed the `pname` to `cmake-format`.